app/js/components/LabResultsList.jsx

```diff
@@ -0,0 +1,263 @@
+  }
+
+  renderLabResultsTable(labResults) {
+    const { dateAndTimeFormat } = this.props;
+    const fields = ["TYPE", "STATUS", "REQUEST DATE", "SAMPLE DATE", "RESULT", "NORMAL RANGE"];
+
+    const columnMetadata = fields.map(columnName => ({
+      Header:
+  <span className={`labs-result-table-head-${columnName.replace(' ', '-').toLocaleLowerCase()}`}>
+    {columnName}
+  </span>,
+      accessor: "",
+      Cell: data => <Cell {...data} columnName={columnName} dateAndTimeFormat={dateAndTimeFormat} type="single" show={false} />,
+      className: `lab-results-list-cell-${columnName.replace(' ', '-').toLocaleLowerCase()}`,
+      headerClassName: `lab-result-list-header-${columnName.replace(' ', '-').toLocaleLowerCase()}`,
+    }));
+    return (
+      <div className="lab-results-list">
+        <SortableTable
+          data={labResults}
+          columnMetadata={columnMetadata}
+          filteredFields={fields}
+          filterType="none"
+          showFilter={false}
+          isSortable={false}
+          rowOnClick={this.handleShowResultsEntryPage}
+          noDataMessage="No orders found"
+          defaultPageSize={10}
+          subComponent={(row) => {
+            const isPanel = (row.original.order.concept.set) && (row.original.status !== "Ordered");
+            const rowFields = ["TYPE", "RESULT", "NORMAL RANGE"];
+            const rowColumnMetadata = rowFields.map(columnName => ({
+              accessor: "",
+              Cell: data => <Cell {...data} columnName={columnName} type="panel" />,
+              className: `lab-results-list-cell-${columnName.replace(' ', '-').toLocaleLowerCase()}`,
+              headerClassName: 'lab-results-list-header',
+            }));
+            if (isPanel) {
+              return (
+                <div className="collapsible-panel">
+                  <SortableTable
+                    data={row.original.encounter.obs[0].groupMembers}
+                    columnMetadata={rowColumnMetadata}
+                    collapseOnDataChange={false}
+                    collapseOnPageChange={false}
+                    showPagination={false}
+                    defaultPageSize={row.original.encounter.obs[0].groupMembers.length}
+                    defaultClassName=""
+                  />
+                </div>
+              );
+            }
+            return '';
+          }}
+        />
+      </div>
+    );
+  }
+
+  render() {
+    const { patients } = this.props;
+    const { patientUUID } = this.state;
+    const selectedPatient = patients[patientUUID] || {};
+    const { encounters = [], orders = [] } = selectedPatient;
+
+    const getPatientLabResults = () => {
+      const labResults = orders.map((order) => {
+        const status = 'Ordered';
+        const matchedEnocunter = encounters.filter((encounter) => {
+          const testOrderObs = encounter.obs.filter(item => item.display.includes('Test order number:'));
```

Comment:
We shouldn't match on display here, as the display may change... we should be able to match on item.concept.uuid I think?